Makefile

```diff
@@ -247,32 +260,34 @@ push: ## Push Grapl containers to Docker Hub
 
 .PHONY: up
 up: build-services ## Build Grapl services and launch docker-compose up
-	$(WITH_LOCAL_GRAPL_ENV) \
+	$(WITH_LOCAL_GRAPL_ENV)
```

Comment:
I presume that the `ONESHELL` trick is what lets you dispense with the `\` escaping here, correct?